site/docs/components/toggle-button/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Toggle Button
+data:
+  description: Toggle Button is a two-state button that the user can toggle. The corresponding action takes effect immediately. With Toggle Button Group, users can make a mutually exclusive selection from a set of related commands—with only one option selected at a time.
+
+  # Fill in the info from the content template's "Metadata" table below.
+  # To omit optional items, comment them out with #
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/main/packages/core/src/toggle-button/ToggleButton.tsx"
+  package:
+    name: "@salt-ds/core"
+  alsoKnownAs: ["Button Group", "Toggle Button Group", "Toggle Group"]
+  relatedComponents: [
+      # Add a { name: "...", relationship: "..." } block for each
+      # related component here (separated by commas).
+      # Permitted values for relationship are: "similarTo" or
+      # "contains".
+      { name: "Checkbox", relationship: "similarTo" },
+      { name: "Switch", relationship: "similarTo" },
+      { name: "Radio Button", relationship: "similarTo" },
+      { name: "Segmented control", relationship: "similarTo" },
```

Comment:
I don't think we have or we will create this component. Was this supposed to be in alsoKnownAs as section maybe?

Reply:
good point - @navkaur76 should we move Segmented control into alsoKnownAs instead as we dont have plans for this component in Salt?